pkg/cmd/server/server.go

```diff
@@ -75,11 +75,18 @@ const (
 	defaultMetricsAddress = ":8085"
 )
 
+type apiConfig struct {
+	backupSyncPeriod, podVolumeOperationTimeout time.Duration
+	restoreResourcePriorities                   []string
+	restoreOnly                                 bool
+}
+
 func NewCommand() *cobra.Command {
 	var (
 		logLevelFlag   = logging.LogLevelFlag(logrus.InfoLevel)
 		pluginDir      = "/plugins"
 		metricsAddress = defaultMetricsAddress
+		config         = apiConfig{}
```

Comment:
I like using a struct here - I think I'd move the above three fields onto it as well